### PR TITLE
Adding new message trigger

### PR DESCRIPTION
**Pull request by DoctorDib**
Changes:
 - Updated message trigger
 - Adding rule methods

NOTE:
This has not been tested as a bot - good luck.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,16 +1,16 @@
 {
-    "name": "screm",
```
Comment (BenRGH) [inline package.json]: Unnecessary file change

File: index.ts [hunk 2]
```diff
@@ -1,52 +1,52 @@
...
+const testRules = ["ping"];
+
+// Checking if string contains any string held within the rules array.
+const ContainsRule = (str: string, rules: string[]) => rules.some(rule => str.toLowerCase().includes(rule));
+
+client.on("ready", () => console.log("Oh I am ready!"));
+
+client.on("messageCreate", (messageEvent:Message) => {
+    const message = messageEvent.content;
+    if (ContainsRule(message, screamRules)) {
```
Comment (BenRGH) [inline index.ts]: Switch statement for future proofing

**Comment by DoctorDib** [comment]
@BenRGH - This has been open for just over a day now, please review.

File: package.json (hunk 1 above)
Comment (DoctorDib) [reply package.json]: Not quite sure what happened there, fixed by indenting.

File: index.ts (hunk 2 above)
Comment (DoctorDib) [reply index.ts]: Outdated

**Review by BenRGH — APPROVED** [review]

**Pull request merged**